tests/atest/rules/misc/empty_variable/expected_output.txt

```diff
@@ -1,14 +1,14 @@
 test.robot:3:1 [I] 0912 Use built-in variable ${EMPTY} instead of leaving variable without value or using backslash
 test.robot:4:1 [I] 0912 Use built-in variable @{EMPTY} instead of leaving variable without value or using backslash
 test.robot:5:1 [I] 0912 Use built-in variable &{EMPTY} instead of leaving variable without value or using backslash
-test.robot:7:3 [I] 0912 Use built-in variable ${EMPTY} instead of leaving variable without value or using backslash
-test.robot:20:3 [I] 0912 Use built-in variable ${EMPTY} instead of leaving variable without value or using backslash
-test.robot:22:3 [I] 0912 Use built-in variable ${EMPTY} instead of leaving variable without value or using backslash
-test.robot:25:3 [I] 0912 Use built-in variable ${EMPTY} instead of leaving variable without value or using backslash
-test.robot:30:3 [I] 0912 Use built-in variable ${EMPTY} instead of leaving variable without value or using backslash
-test.robot:33:3 [I] 0912 Use built-in variable ${EMPTY} instead of leaving variable without value or using backslash
-test.robot:35:25 [I] 0912 Use built-in variable ${EMPTY} instead of leaving variable without value or using backslash
```

Comment:
So now we're always pointing at the start of the line rather than place where the 'empty' variable is? 

Ie
```
${EMPTY_WITH_BACKSLASH}  \
```
previous showed at "\" and now shows at "$"

Reply:
Yes, because from the user's perspective, there is a problem with this whole variable, so pointing to a place where there is no value (it basically underlined empty space before) is hard to notice in the IDE.